### ISSUE TITLE
[accessibility] Projector tab has no lang or xml:lang attribute

### ISSUE THREAD

**Issue by jayfurmanek** [tensorflow/tensorboard#3189]
## Issue description

I've been running some accessibility assessment tools against Tensorboard. This issue describes a violation discovered by IBM Dynamic Assessment Plugin with Policy level: Accessibility [November 2019] which contains the WCAG 2.1 ruleset.

Frame: TensorBoard
Page : http://localhost:6006/#projector&run=.
Issue: Page detected as HTML, but does not have a lang attribute.
Fix: Use the language attribute on HTML elements

Identify the default language of the document by providing the lang or xml:lang attribute on the HTML element.

Rationale

Speech synthesizers that support multiple languages can adapt to the pronunciation and syntax that are specific to the language, speaking the text in the correct language with proper pronunciation. Some screen readers can be configured to automatically switch languages based on the lang attribute; for example: <html lang=“en">.

### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version d515ab103e2b1cfcea2b096187741a0eeb8822ef

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='04c062ec65ed', release='4.14.0-115.16.1.el7a.ppc64le', version='#1 SMP Wed Nov 27 18:01:42 UTC 2019', machine='ppc64le')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: True
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==1.15.0
INFO: installed: tensorflow==1.15.0
INFO: installed: tensorflow-estimator==1.15.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '1.15.0'

--- check: tensorflow_python_version
2020-01-28 21:50:38.452524: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
INFO: tensorflow.__version__: '1.15.0'
INFO: tensorflow.__git_version__: 'v1.15.0-20-g3396836df0'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/opt/anaconda3/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 32>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 32>
Loopback infos: [(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0)), (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): '04c062ec65ed'

--- check: stat_tensorboardinfo
INFO: directory: /tmp/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=3278538768, st_dev=161, st_nlink=2, st_uid=1084, st_gid=1500, st_size=6, st_atime=1574193500, st_mtime=1578691457, st_ctime=1578691457)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/opt/anaconda3/lib/python3.7/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.7.1
alabaster==0.7.12
anaconda-client==1.7.2
anaconda-project==0.8.2
asn1crypto==0.24.0
astor==0.7.1
astroid==2.2.5
astropy==3.0.5
atomicwrites==1.3.0
attrs==19.1.0
Babel==2.6.0
backcall==0.1.0
backports.os==0.1.1
backports.shutil-get-terminal-size==1.0.0
beautifulsoup4==4.7.1
bitarray==0.8.3
bkcharts==0.2
bleach==3.1.0
bokeh==1.0.4
boto==2.49.0
Bottleneck==1.2.1
certifi==2019.9.11
cffi==1.12.2
chardet==3.0.4
Click==7.0
cloudpickle==0.8.0
clyent==1.2.2
colorama==0.4.1
conda==4.6.14
conda-build==3.16.3
conda-verify==3.1.1
contextlib2==0.5.5
cryptography==2.6.1
cycler==0.10.0
Cython==0.29.6
cytoolz==0.9.0.1
dask==1.1.4
decorator==4.4.0
defusedxml==0.5.0
distributed==1.26.0
docutils==0.14
entrypoints==0.3
et-xmlfile==1.0.1
fastcache==1.0.2
filelock==3.0.10
Flask==1.0.2
future==0.17.1
gast==0.2.2
gevent==1.4.0
glob2==0.6
gmpy2==2.0.8
google-pasta==0.1.6
graphsurgeon==0.4.1
greenlet==0.4.15
grpcio==1.16.1
h5py==2.8.0
heapdict==1.0.0
html5lib==1.0.1
idna==2.8
imageio==2.5.0
imagesize==1.1.0
importlib-metadata==0.0.0
ipykernel==5.1.0
ipython==7.4.0
ipython-genutils==0.2.0
ipywidgets==7.4.2
isort==4.3.16
itsdangerous==1.1.0
jdcal==1.4
jedi==0.13.3
Jinja2==2.10
joblib==0.14.0
jsonschema==3.0.1
jupyter==1.0.0
jupyter-client==5.2.4
jupyter-console==6.0.0
jupyter-core==4.4.0
jupyterlab==0.35.4
jupyterlab-server==0.2.0
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.0.1
lazy-object-proxy==1.3.1
libarchive-c==2.8
locket==0.2.0
lxml==4.3.2
Markdown==3.1.1
MarkupSafe==1.1.1
matplotlib==3.0.3
mccabe==0.6.1
mistune==0.8.4
more-itertools==6.0.0
mpmath==1.1.0
msgpack==0.6.1
multipledispatch==0.6.0
nbconvert==5.4.1
nbformat==4.4.0
networkx==2.2
nltk==3.4
nose==1.3.7
notebook==5.7.8
numexpr==2.7.0
numpy==1.16.5
numpydoc==0.8.0
olefile==0.46
packaging==19.0
pandas==0.23.4
pandocfilters==1.4.2
parso==0.3.4
path.py==11.5.0
pathlib2==2.3.3
patsy==0.5.1
pep8==1.7.1
pexpect==4.6.0
pickleshare==0.7.5
Pillow==6.2.1
pip==19.0.3
pkginfo==1.5.0.1
pluggy==0.9.0
ply==3.11
prometheus-client==0.6.0
prompt-toolkit==2.0.9
protobuf==3.8.0
psutil==5.5.0
ptyprocess==0.6.0
py==1.8.0
pycodestyle==2.5.0
pycosat==0.6.3
pycparser==2.19
pycrypto==2.6.1
pycurl==7.43.0.2
pyflakes==2.1.1
Pygments==2.3.1
pylint==2.3.1
pyodbc==4.0.26
pyOpenSSL==19.0.0
pyparsing==2.3.1
pyrsistent==0.14.11
PySocks==1.6.8
pytest==4.3.1
pytest-arraydiff==0.3
pytest-astropy==0.5.0
pytest-doctestplus==0.3.0
pytest-openfiles==0.3.1
pytest-remotedata==0.3.1
python-dateutil==2.8.0
pytz==2018.9
PyWavelets==1.0.2
PyYAML==5.1
pyzmq==18.0.0
requests==2.21.0
ruamel-yaml==0.15.46
scikit-image==0.14.1
scikit-learn==0.21.3
scipy==1.3.0
seaborn==0.9.0
Send2Trash==1.5.0
setuptools==41.0.1
simplegeneric==0.8.1
singledispatch==3.4.0.3
six==1.12.0
snowballstemmer==1.2.1
sortedcollections==1.1.2
sortedcontainers==2.1.0
soupsieve==1.8
Sphinx==1.8.5
sphinxcontrib-websupport==1.1.0
SQLAlchemy==1.3.1
statsmodels==0.9.0
sympy==1.3
tables==3.4.4
tblib==1.3.2
tensorboard==1.15.0
tensorflow==1.15.0
tensorflow-estimator==1.15.1
tensorrt==6.0.1.5
termcolor==1.1.0
terminado==0.8.1
testpath==0.4.2
tflms==2.0.2
toolz==0.9.0
tornado==6.0.2
tqdm==4.31.1
traitlets==4.3.2
uff==0.6.5
unicodecsv==0.14.1
urllib3==1.24.1
wcwidth==0.1.7
webencodings==0.5.1
Werkzeug==0.15.4
wheel==0.33.1
widgetsnbextension==3.4.2
wrapt==1.11.1
xlrd==1.2.0
XlsxWriter==1.1.5
xlwt==1.3.0
zict==0.1.4
zipp==0.3.3

``````

</details>

### Next steps

No action items identified. Please copy ALL of the above output,
including the lines containing only backticks, into your GitHub issue
or comment. Be sure to redact any sensitive information.

**Comment by wchargin** [comment]
Hi @jayfurmanek! Thanks for the report. Sure, we can go ahead and add an
`<html lang="en">` to the main page (which will affect all plugins).
